src/style.css

```diff
@@ -1,3 +1,75 @@
-body {
+html {
+  font-family: Roboto, sans-serif;
+
+  --text-color:#000;
+  --hover-color:#00ACDC;
+  ;
+}
+
+body, div, p, h1, h2, h3,a,li,ul,nav {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100vw;
+}
+
+.container {
+  margin: 0 50px;
+  display: flex;
+  padding: 10px 0;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  display: flex;
+}
+
+.navorder {
+  display: flex;
+  list-style: none;
+}
+
+.navelement > a {
```

Comment:
[Don't use tag names for styling (except html and body)](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#css)